### PR TITLE
#36 -  Life as a Torchboxer

### PR DESCRIPTION
**Pull request by jhancock532**
[Ticket on Codebase](https://projects.torchbox.com/projects/contentful-careers-page/tickets/36).

This component copies styling code from previous PR, for example the link utility class in #18. The title and links are intentionally hardcoded, the existing `children` passed to the component will become a `RichText` element.

Instead of screenshots, please view the demo deployment linked below (saving bandwidth and space).

### CODE REVIEW

File: styles/LifeAsATorchboxer.module.scss [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    text-align: left;
+    margin-bottom: $grid;
+    order: 1;
+
+    @include media-query(medium) {
+        order: 2;
+    }
+
+    @include media-query(large) {
+        max-width: 635px;
```
Comment (Morsey187) [inline styles/LifeAsATorchboxer.module.scss:26]: **Question:** This looks more like 540px in the designs, was there a reason you added this size?

File: styles/LifeAsATorchboxer.module.scss [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+.richText {
+    & > p {
+        color: var(--color--dark-indigo);
+    }
+}
+
+.links {
+    @include font-size(m);
+    & a:first-child {
```
Comment (Morsey187) [inline styles/LifeAsATorchboxer.module.scss:52]: **Thought:** I think there would usually be a space abover this line, would be good to make another ticket to sort out formatting.

File: styles/_utilities.scss [hunk 3]
```diff
@@ -0,0 +1,55 @@
+.underline-link {
+    font-weight: $weight--bold;
+
```
Comment (Morsey187) [inline styles/_utilities.scss:3]: Again formatting looks to be a little off, might be worth doing this in another ticket when theres less open MRs with css properties

**Review by Morsey187 — CHANGES_REQUESTED** [review]
Looks great just one question about the site width which would be good to tackle in this MR

File: styles/LifeAsATorchboxer.module.scss [hunk 4]
```diff
@@ -0,0 +1,76 @@
+.container {
```
[the remark this answers is not in the document]
Comment (jhancock532) [reply styles/LifeAsATorchboxer.module.scss:1]: I will do further homepage polishing in a future ticket, added these suggestions for now 👍

File: styles/LifeAsATorchboxer.module.scss (hunk 1 above)
Comment (jhancock532) [reply styles/LifeAsATorchboxer.module.scss:26]: This text container encompasses the title (540px) and the description text (635px)

File: styles/_utilities.scss (hunk 3 above)
Comment (jhancock532) [reply styles/_utilities.scss:3]: Sounds good, will double check the stylelint setup

**Pull request merged**